documentation/my-homeworks/bulletin-board/src/Board.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react'
+import Note from './Note'
+import FaPlus from 'react-icons/lib/fa/plus'
+
+class Board extends Component {
+	constructor(props) {
+		super(props)
+		this.state = {
+			notes: []
+		}
+		this.add = this.add.bind(this)
+		this.eachNote = this.eachNote.bind(this)
+		this.update = this.update.bind(this)
+		this.remove = this.remove.bind(this)
+		this.nextId = this.nextId.bind(this)
+	}
+
+	componentWillMount() {
+		var self = this
+		if(this.props.count) {
+			fetch(`https://baconipsum.com/api/?type=all-meat&sentences=${this.props.count}`)
+				.then(response => response.json())
+				.then(json => json[0]
+								.split('. ')
```

Comment:
This long white space usually happens because you are using *tabs* instead of *spaces* on your editor, so please stick to *spaces*.